src/test/java/uk/gov/pay/directdebit/mandate/dao/MandateDaoIT.java

```diff
@@ -210,4 +213,58 @@ public void shouldUpdateReferenceAndReturnNumberOfAffectedRows() {
         assertThat(mandateAfterUpdate.get("state"), is(testMandate.getState().toString()));
         assertThat(mandateAfterUpdate.get("type"), is(testMandate.getType().toString()));
     }
+
+    @Test
+    public void findAllMandatesBySetOfStatesAndMaxCreationTime_shouldNotFindMandate_WrongState() {
```

Comment:
could this method be shortened, e.g. `shouldNotFindMandateInTheWrongState`?

Reply:
Thanks, done.